workers/unity/Packages/io.improbable.worker.sdk/CIO.cs

```diff
@@ -0,0 +1,178 @@
+using System.Runtime.InteropServices;
+using Improbable.Worker.CInterop.Internal;
+using Int64 = System.Int64;
+using Uint64 = System.UInt64;
+using Uint32 = System.UInt32;
+using Uint8 = System.Byte;
+using Char = System.Char;
+using IntPtr = System.IntPtr;
+
+namespace Improbable.Gdk
+{
+    internal unsafe class CIO
+    {
+#if UNITY_IOS
+        private const string WorkerLibrary = "__Internal";
+#else
+        private const string WorkerLibrary = "improbable_worker";
+#endif
```

Comment:
We could put this in its own file and reference it both here and in `CEventTrace.cs`